library/rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date:, book:, person:)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
You are missing two aspects of the association in this constructor. Let's add them

- [ ] The rental is adding itself to the @book achieving the 'belongs-to' association
    `book.rentals << self`
- [ ] The rental is adding itself to the @person the 'belongs-to' association
    `person.rentals << self`